tools/toolchains/arm.py

```diff
@@ -160,9 +160,9 @@ def version_check(self):
 
     def _get_toolchain_labels(self):
         if getattr(self.target, "default_toolchain", "ARM") == "uARM":
-            return ["ARM", "ARM_MICRO"]
+            return ["ARM", "ARM_MICRO", "ARMC5"]
```

Comment:
What does it do?

Reply:
Means we can optionally include stuff via a `TOOLCHAIN_ARMC5` folder. At the minute we only have the labels `TOOLCHAIN_ARM` and `TOOLCHAIN_ARMC6`. Used for the base `<array>`, `<initializer_list>` bits for ARM C 5.

(We only need to tiptoe around with namespace `mstd` and `<mstd_xxx>` for places where we are up against an existing header file; when we're supplying something ARM C 5 lacks altogether, we can just go straight in with the standard name in namespace `std`.)